performance_test/measure_time.py

```diff
@@ -30,8 +30,8 @@ def measure(self, game_field, kbdinput):
         shutil.copy('./performance_test/' + game_field, 'current_state.txt')
         print('field = {:<15s} key = {:5s}    measuring '.format(
             game_field, kbdinput),
-              end='',
-              flush=True)
+            end='',
+            flush=True)
```

Comment:
I check PEP8 compliance with `flake8` before committing, I still missed some spaces on 2 lines that autopep rectified. I learned the lesson and installed autopep now :-)